### PR TITLE
[IMP:Transfer details config only for admin]

### PR DESCRIPTION
**Pull request by rmu-odoo**
Hello,

Task:https://www.odoo.com/web#id=12785&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.s3IUEUphWgi6lBfP

With this PR admin can reset the bank account number and communition details for particular wire transfer.

Thanks.

### CODE REVIEW

File: addons/website_sale/views/templates.xml [hunk 1]
```diff
@@ -1346,7 +1346,12 @@
     <t t-if="tx_ids and state == 'pending' and validation == 'manual'">
       <p>Your transaction is waiting a manual confirmation.</p><br />
       <t t-if="tx_post_msg">
-        <div class="hidden-print"><t t-raw="tx_post_msg"/></div>
+        <div class="hidden-print">
+          <span groups='base.group_system'>
+             <a class="btn btn-primary pull-right" target="_blank" t-att-href="'/web#return_label=Website&amp;model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % ('payment.acquirer',order.payment_acquirer_id.id, 'payment.action_payment_acquirer')"><i class="fa fa-cog"></i> Configure Transfer Details</a>
```
Comment (tivisse) [inline addons/website_sale/views/templates.xml]: Why return_label=Website ?

File: addons/website_sale/controllers/main.py [hunk 2]
```diff
@@ -872,6 +872,7 @@ def payment_get_status(self, sale_order_id, **post):
                 values.update({
                     'tx_ids': True,
                     'state': state,
+                    'order': order,
```
Comment (tde-banana-odoo) [inline addons/website_sale/controllers/main.py]: Why this change ?

File: addons/website_sale/views/templates.xml (hunk 1 above)
Comment (tde-banana-odoo) [inline addons/website_sale/views/templates.xml]: +1

File: addons/website_sale/controllers/main.py (hunk 2 above)
Comment (rmu-odoo) [inline addons/website_sale/controllers/main.py]: @yti : yes , you are right . we are redireting it to backend. not to website. i wil make that change.
Thanks.

Comment (rmu-odoo) [inline addons/website_sale/controllers/main.py]: @tde: redirect  into that order's particular payment_acquirer_id. So user dont have to find that wire transfer from list. it will redirect to the chosen wire transfer form.
Thanks

Comment (hmo-odoo) [inline addons/website_sale/controllers/main.py]: @rmu-odoo 

I think, you need acquirer_id instead of order object. so just pass acquirer_id into template.
